### PR TITLE
start implement rename

### PR DESCRIPTION
**Pull request by erwan-joly**

### CODE REVIEW

**Comment by erwan-joly** [comment]
https://github.com/NosCoreIO/NosCore/issues/997

**Comment by 0Lucifer0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/NosCore.Controllers/CharacterScreen/CharRenPacketHandler.cs  8
- test/NosCore.Tests/PacketHandlerTests/BiPacketHandlerTests.cs  1
- src/NosCore.Core/HttpClients/MasterServerHttpClient.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5496257&bid=18011210)

File: src/NosCore.Controllers/CharacterScreen/CharNewPacketHandler.cs [hunk 1]
```diff
@@ -41,6 +41,8 @@ public class CharNewPacketHandler : PacketHandler<CharNewPacket>, IWorldPacketHa
         private readonly IDao<CharacterDto, long> _characterDao;
         private readonly IDao<MinilandDto, Guid> _minilandDao;
 
+        public const string Nameregex =
```
Comment (0Lucifer0) [inline src/NosCore.Controllers/CharacterScreen/CharNewPacketHandler.cs:44]: Codacy Issue found: [Change this constant to a 'static' read-only property.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5496257)

**Comment by 0Lucifer0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/NosCore.Controllers/CharacterScreen/CharRenPacketHandler.cs  8
- src/NosCore.Core/HttpClients/MasterServerHttpClient.cs  1
- test/NosCore.Tests/PacketHandlerTests/BiPacketHandlerTests.cs  1
- test/NosCore.Tests/PacketHandlerTests/CharRenPacketHandlerTests.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5496257&bid=18011210)

File: src/NosCore.Controllers/CharacterScreen/CharNewPacketHandler.cs (hunk 1 above)
Comment (0Lucifer0) [inline src/NosCore.Controllers/CharacterScreen/CharNewPacketHandler.cs:44]: Codacy Issue found: [Change this constant to a 'static' read-only property.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5496257)

**Pull request merged**